src/components/misc/ConnectionList.jsx

```diff
@@ -18,9 +17,8 @@ export default class ConnectionList extends React.Component {
         else {
             let list = this.props.connectionList;
             let items;
-
-            if (list.get('items').size) {
-                items = list.get('items').toList().map(i => {
+            if (list.size) {
+                items = list.toList().map(i => {
```

Comment:
Why this change? It seems to break this component, which is now throwing an error while rendering because `list.toList()` is not the items of the list, but the properties/values of the list, e.g. `isPending`/`false`. The first element is not a map representing a connection, but is instead `false`, which does not have a `.get()` function, so the next line raises an exception.

Reply:
This was due to a change on line 22 of OrgSectionPage, where connections are filtered to only the followed. It works on the following OrgSectionPage, but not on the SettingsPage.

I don't recall because this was such a long time ago and I must have missed the SettingsPage when testing it more recently as well, but I think this was written before the following endpoint was implemented.

It has now been updated to use the following endpoint.